### PR TITLE
Extend badges in readme and add logo

### PR DESCRIPTION
**Pull request by davidrudlstorfer**
## Description and Context

Added the current 4C logo as a replacement to the name 4C in the header

Also saved the static badges into assets and added them to the readme

## Related Issues and Merge Requests

#42

### CODE REVIEW

**Comment by davidrudlstorfer** [comment]
image

**Comment by davidrudlstorfer** [comment]
@amgebauer the github logo is still pulled from github within the SVG file for the badges - we could also remove the Github logo

**Comment by sebproell** [comment]
> @amgebauer the github logo is still pulled from github within the SVG file for the badges - we could also remove the Github logo

I think we can just drop the logo here although I guess pulling from Github is OK since we are on Github anyway. The reason why I would drop it is that these badges are not really related to Github. We self-host the website and the docs are separate from the repo as well. Yes, we create them via actions, but the badge is not really about the development on Github which is the case for the workflow badges.

**Comment by davidrudlstorfer** [comment]
> > @amgebauer the github logo is still pulled from github within the SVG file for the badges - we could also remove the Github logo
> 
> I think we can just drop the Github logo here although I guess pulling from Github is OK since we are on Github anyway. The reason why I would drop it is that these badges are not really related to Github. We self-host the website and the docs are separate from the repo as well. Yes, we create them via actions, but the badge is not really about the development on Github which is the case for the workflow badges.

Done ✅

File: assets/logo/4c-logo.svg
Comment (amgebauer) [inline assets/logo/4c-logo.svg:1]: I'm not sure whether we want to add this logo now. We never agreed on a logo yet. Maybe just leave the 4C title for now.

**Comment by davidrudlstorfer** [comment]
@sebproell I've moved the `assets` into `utilities`

@amgebauer as discussed the logo is identical to the website, once a new logo is designed both need to be updated

**Comment by mayrmt** [comment]
I appreciate the initiative, but there is no official logo yet, so please do not put one into the repository. Thanks!

**Review by mayrmt — CHANGES_REQUESTED** [review]
I appreciate the initiative, but there is no official logo yet, so please do not put one into the repository. Thanks!

(Yes, there's a logo on the website. In my opinion, it should not be there either.)

**Comment by davidrudlstorfer** [comment]
@mayrmt thanks for your feedback 🙏🏻

I've removed the logo from the readme as well as from the utilities and adjusted the commit message accordingly.

To stay consistent between the Github repo and our website we should also remove the logo from the website ASAP and create an issue that we need a logo for the usage within the repo and on the website. Maybe someone can remove the logo from the website

Therefore, this MR would be ready for review again

**Review by mayrmt — APPROVED** [review]

**Review by amgebauer — APPROVED** [review]

**Pull request merged**